.github/workflows/gh-pages.yaml

```diff
@@ -21,10 +23,14 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version: '1.18'
-      - run: pip install mkdocs==1.2.4 mkdocs_material==8.1.9
-      - run: mkdocs build
-      - run: make parse-examples
+      - run: make docs
```

Comment:
use the same `make` target both locally and on PR to make it easier to debug CI failure